ERROR - 2022-05-12 11:08:04,626 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:08:09,814 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:08:10,982 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:08:47,923 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:08:50,574 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:08:52,364 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:12:54,915 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:15:05,850 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:15:08,649 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:18:03,147 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:18:05,076 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:18:07,967 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:18:09,214 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:18:11,253 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:18:13,204 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:20:18,336 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
ERROR - 2022-05-12 11:20:31,487 - process: 40044 - utils.py - polls.utils - 76 - utils - [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
Traceback (most recent call last):
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 74, in wrapper
    result = func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\vr_env\lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\views.py", line 414, in project_tree
    tree = get_tree(join(path, project_name))
  File "C:\Users\46121\PycharmProjects\CS65_3\cs65_4\cs65_5\polls\utils.py", line 101, in get_tree
    for file in os.listdir(path):
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\46121\\PycharmProjects\\CS65_3\\cs65_4\\cs65_5\\projects\\undefined'
